docs/landing/project/README.md

```diff
@@ -0,0 +1,54 @@
+# Project
+
+## Video demo
+
+[![See video demo!](https://divante.com/github/shopware-pwa/shopware-pwa-miniature.png)](https://youtu.be/t2JJgvvxMG8)
+
+Shopware PWA is a project created in a cooperation between [Vue Storefront](https://www.vuestorefront.io/) and [shopware AG](https://shopware.com/) in order to deliver a state-of-the-art customisable frontend for Shopware 6.
+
+It takes the capabilities of Shopware 6 and brings it to the hands of the frontend developers to create outstanding customer experiences based on modern frontend technologies like:
+
+ * Vue.js
+ * Nuxt.js
+ * Typescript
+ * Jest
+ * and many more
+
+Many of the Shopware Core concepts you are used to will be available as well, including:
+
+ * Shopping Experiences (CMS)
+ * Search and filters
+ * Product variants
+ * Checkout and payment
+ * User self-management
+ * Plugin system
+
+If you have a specific set of features, feel free to compare it against our curated and regularly updated [Feature List](/guide/featurelist)
+
+## Why a new storefront?
+
+Good question - Shopware already comes with a storefront. However, it is a headless commerce platform which means that it has no "native" frontend to it. So even the **default** Storefront is just **one** frontend that can be used.
+
+We are observing a paradigm shift that lets more and more projects and customers transition into more universal frontend applications based on Javascript. They have the great benefit to integrate easily with other services like CMS, personalisation engines or search by sharing code on both front- and backend.
+
+The full frontend stack can be Javascript based and, depending on your requirements, use features like SEO-friendly server-side rendering, smart pre-compilation or reactive elements and PWA. It can also provide fully static pages if you wish so.
```

Comment:
```suggestion
The full frontend stack can be JavaScript based and, depending on your requirements, use features like SEO-friendly server-side rendering, smart pre-compilation or reactive elements and PWA. It can also provide fully static pages if you wish so.
```